source/chapters/library.rst

```diff
@@ -313,10 +314,12 @@ Tracks can be loaded into a deck in several ways:
   load a track into a deck.
 * By :ref:`control-keyboard` to load the selected track in the library track
   table.
+* Using keyboard :term:`shortcut <shortcuts>` keys to load track in Deck 1 with :kbd:`Shift` + :kbd:`Left` and Deck 2
+  with :kbd:`Shift` + :kbd:`Right`
```

Comment:
<shortcuts> reference doesn't exist.
Please merge this and the above bullet point, it looks redundant right now:
![image](https://user-images.githubusercontent.com/5934199/96645925-9b013080-132b-11eb-82dd-875c42c59310.png)
Something like this: "Use a keybord(reflink): 'Up' and 'Down' to select a track, Shift+Left to load it to deck 1 ....."

Reply:
So this can be:
* Use a keyboard: "Up" and "Down" arrow keys to select a track, Shift + left to load track in Deck 1 and Shift + Right to load track in Deck 2

Reply:
Please remove the change.
I just noticed this is from #135 but I don't see your other changes are based on that PR.
(and I move my review to #135 )

Reply:
Ok, I'll remove that and just mention the properties. Thank you